docs: Add FAQ page for the Dev Toolbar (#12182)
getsentry/sentry-docs: docs/product/dev-toolbar/faq.mdx

---
title: FAQ
sidebar_order: 30
description: "Frequently asked questions about the Dev Toolbar."
---

<Expandable title="Are there plans to include the Dev Toolbar in the JavaScript SDK?">

The the Dev Toolbar is very distinct from the JavaScript SDK and will not be merged into the [JavaScript SDK](https://github.com/getsentry/sentry-javascript).

Some of the differences include:

>   - The toolbar has a different set of [dependencies](https://github.com/getsentry/sentry-toolbar/blob/main/package.json) and uses different browser API's that the JavaScript SDK does not use. For example: the toolbar will interact with things like cookies and localstorage. By keeping these pieces of code separate it is easier to audit the [SDK code on github](https://github.com/getsentry/sentry-javascript) to verify that it is not persisting information inside end-users browsers.

Reviewer: ```suggestion
  - The toolbar has a different set of [dependencies](https://github.com/getsentry/sentry-toolbar/blob/main/package.json) and uses different browser APIs that the JavaScript SDK does not use. For example: the toolbar will interact with things like cookies and local storage. By keeping these pieces of code separate, it is easier to audit the [SDK code on GitHub](https://github.com/getsentry/sentry-javascript) to verify that it is not persisting information inside end-users' browsers.
```

---
title: FAQ
sidebar_order: 30
description: "Frequently asked questions about the Dev Toolbar."
---

<Expandable title="Are there plans to include the Dev Toolbar in the JavaScript SDK?">

> The the Dev Toolbar and the [JavaScript SDK](https://github.com/getsentry/sentry-javascript) are distinct features that we intentionally keep separated.

Author: ```suggestion
The Dev Toolbar and the [JavaScript SDK](https://github.com/getsentry/sentry-javascript) are distinct features that we intentionally keep separated.
```

> The the Dev Toolbar is very distinct from the JavaScript SDK and will not be merged into the [JavaScript SDK](https://github.com/getsentry/sentry-javascript).

Reviewer 2: ```suggestion
The the Dev Toolbar and the [JavaScript SDK](https://github.com/getsentry/sentry-javascript) are distinct features that we intentionally keep separated.
```

> Some of the differences include:

Reviewer 2: ```suggestion
Some of the differences between the two include:
```

  - The toolbar is a UI product focused on making it easier to find and take action on existing data, while the SDK functions as infrastructure to collect and send data to the server.

>   - The toolbar has a different set of [dependencies](https://github.com/getsentry/sentry-toolbar/blob/main/package.json) and uses different browser APIs that the JavaScript SDK does not use. For example: the toolbar will interact with things like cookies and local storage. By keeping these pieces of code separate, it is easier to audit the [SDK code on GitHub](https://github.com/getsentry/sentry-javascript) to verify that it is not persisting information inside end-users' browsers.

Reviewer 2: ```suggestion
  - The toolbar has a different set of [dependencies](https://github.com/getsentry/sentry-toolbar/blob/main/package.json) and uses different browser APIs that the JavaScript SDK does not use. For example: the toolbar will interact with things like cookies and local storage. By keeping these pieces of code separate, it's easier to audit the [SDK code on GitHub](https://github.com/getsentry/sentry-javascript) to verify that it is not persisting information inside end-users' browsers.
```

>   - The setup & deploy instruction are very different. The SDK is best deployed on staging and production environments, and can be configured easily with environment variables. The Dev Toolbar requires special considerations to deploy it into production, usually by creating a condition so that it's only included for members of your own Sentry organization. This is why it's important to keep the setup instructions separate between the two.

Reviewer 2: ```suggestion
  - The setup and deploy instruction are very different. The SDK is best deployed on staging and production environments, and can be configured easily with environment variables. The Dev Toolbar requires special considerations to deploy it into production, usually by creating a condition so that it's only included for members of your own Sentry organization. This is why it's important to keep the setup instructions separate between the two.
```
